handlers/lobbies.js

```diff
@@ -45,12 +45,13 @@ Dota2.Dota2Client.prototype.createTournamentLobby = function(password, tournamen
     if (this.debug) util.log("Sending match CMsgPracticeLobbyCreate request");
     var lobby_details = Dota2._parseOptions(options, Dota2._lobbyOptions);
     lobby_details.pass_key = password;
+    lobby_details.leagueid = options.leagueid || tournament_id;
```

Comment:
Seems like the issue was being caused by this. After all, `createPracticetLobby` calls `createTournamentLobby`. So if the user doesn't set the option `leagueid` but does pass the parameter `tournament_id`, `leagueid` should be `tournament_id`.